Minor clean up in life_of.md (#1608)
gpuweb/cts: docs/intro/life_of.md

Anyone with access should do this eagerly, to ensure no testing ideas are forgotten.
(And if you don't have access, just file a regular issue.)

> 1. Enter a draft issue with "Status: New (not in repo)" in the
>     [CTS project tracker](https://github.com/orgs/gpuweb/projects/3) with any available info

Reviewer: Since the string "Status: New (not in repo)" doesn't show up anywhere else I wouldn't quote it all together. What do you think of:

```suggestion
1. Enter a [draft issue](https://github.com/orgs/gpuweb/projects/3), with the Status
    set to "New (not in repo)", and any available info included in the issue description
```

Author: Looking nice, thanks for the suggestion
